e2e/src/app.po.ts

```diff
@@ -32,17 +32,17 @@ export class AppPage {
 		return el;
 	}
 
-	getHtmlString( el: WebElement ) {
+	async getHtmlString( el: WebElement ) {
 		return el.getAttribute( 'innerHTML' ).then( str => str.replace( /\u200B/g, '' ) );
```

Comment:
`.then()` method part can be removed here since we are no longer sending a non-breaking space in the test.